lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -410,6 +411,25 @@ def add_check_manifest_lock_script_phase
           end
         end
 
+        # Adds a symlink at $PROJECT_DIR/Pods pointing to $PODS_ROOT if the user project is behind a symlink
+        # if the user project is a symlink
```

Comment:
The second line `if the user project is a symlink` is unneccessary

Reply:
done